chapters/connectors.tex

```diff
@@ -903,24 +905,58 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
+When generating connection graph equations, only the overdetermined components of a connector are considered.
+The connection graph equations replace the equality-equations for variables that are neither flow nor stream in \cref{generation-of-connection-equations}.
+
+\subsubsection{Remove Redundant Connect-Equations}
+In order to eliminate any redundant \lstinline!connect!-equation the following preparation is needed.
+\begin{nonnormative}
+In the common case where there are no redundant \lstinline!connect!-equations a \lstinline!connect! between connectors with \emph{one} overdetermined component will directly generate \emph{one} optional spanning-tree edge.
+\end{nonnormative}
+\begin{enumerate}
+\item
+  The connection sets are built similarly to the normal way, but keeping the overdetermined components as primitives.
+\item
+  Instead of generating the equality-equation for an overdetermined component, an optional spanning-tree edge in the virtual connection graph is constructed.
+\end{enumerate}
```

Comment:
(Attaching comment to vacant line above where this actually belongs.)
It's not really possible to count redundant `connect`-equations like it's done below.  That there's an `connect`-equation redundancy of `m - (n - 1)` doesn't mean that exactly `m - (n - 1)` `connect`-equations can be identified as the redundant ones.  I think this can be resolved by talking about the _connect-equation redundancy_ instead.  I'll show with a series of detailed suggestions below…

Reply:
I understand that in general it doesn't follow, but apart from the fact that each connect-equation involves more than the overdetermined component, I don't see how there can be a connection-redundancy that cannot be (ambiguously) linked to specific connect-equations.